### PR TITLE
Updates to assembly tutorial for large genome

### PR DESCRIPTION
**Pull request by AnnaSyme**
Two commits - one for user to check input datatypes, and one to recommend using either of Galaxy Eu or Au servers.

### CODE REVIEW

File: topics/assembly/tutorials/largegenome/tutorial.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ contributors:
 
 # Introduction
 
+*Note: We recommend running this tutorial on either the Galaxy Europe or Galaxy Australia servers. Other servers (such as Galaxy main) have not yet been configured fully for all the tools in this analysis.*
```
Comment (hexylena) [inline topics/assembly/tutorials/largegenome/tutorial.md:28]: If this note is specific to Smörgåsbord, I don't think we need to include it. Under every video we'll shown a list of which servers are supported (we check every tutorial against every tool version that's installed on each server.)

**Comment by hexylena** [comment]
There's a linting error where you've skiped a heading level, could I trouble you to check into that?

File: topics/assembly/tutorials/largegenome/tutorial.md (hunk 1 above)
Comment (AnnaSyme) [reply topics/assembly/tutorials/largegenome/tutorial.md:28]: Thanks @hexylena - sounds good, yes let's not include this.

**Comment by AnnaSyme** [comment]
> There's a linting error where you've skiped a heading level, could I trouble you to check into that?

Yes, thanks, have fixed this in a new commit.

**Review by hexylena — APPROVED** [review]

**Pull request merged**